### ISSUE TITLE
Is there a batch_norm layer in your "example/alexnet.yaml"

### ISSUE THREAD

**Issue by Jokeren** [NervanaSystems/neon#241]
I read the file, and I find there's an annotation that tells me there's a `batch_norm` layer. However, according to the code, you haven't explicitly declare the `batch_norm` variable. Therefore, it seems that the `batch_norm` layer will not be initialized by default.

**Comment by apark263** [comment]
Yes you are correct that the comment indicates batch norm and bias but only has bias in the actual network. We will edit that comment.

**Comment by jennifermyers** [comment]
The fix is lined up for v1.5.0 release.
